### PR TITLE
More Android work

### PR DESCRIPTION
**Pull request by kumpera**

### CODE REVIEW

File: realm/realm.js [hunk 1]
```diff
@@ -47,25 +50,69 @@ Symptoms.schema = {
   },
 };
 
+const REAL_PW_KEY = "REALM_PW_KEY";
```
Comment (cheng-tan) [inline realm/realm.js]: REALM_PW_KEY?

File: realm/realm.js [hunk 2]
```diff
@@ -47,25 +50,69 @@ Symptoms.schema = {
...
+const REAM_KEY_SIZE = 64;
```
Comment (cheng-tan) [inline realm/realm.js]: REALM_KEY_SIZE?

File: realm/realm.js [hunk 3]
```diff
@@ -47,25 +50,69 @@ Symptoms.schema = {
...
+let createKeyInstance = null;
 const getKey = async () => {
   try {
-    const keyString = await NativeModules.EncryptionUtil.getRealmKey();
+    let keyString = null;
+    
+    if(Platform.OS == 'ios') {
+      keyString = await NativeModules.EncryptionUtil.getRealmKey();
+    } else {
+      //this strings won't show up since they are only used if biometric auth is used (which isn't here)
```
Comment (cheng-tan) [inline realm/realm.js:71]: Can this be used for iOS?

File: realm/realm.js (hunk 1 above)
Comment (kumpera) [reply realm/realm.js]: Will fix the typos.

File: realm/realm.js (hunk 3 above)
Comment (kumpera) [reply realm/realm.js:71]: Probably, but I haven't tested it there.

**Pull request merged**